packages/runtime-dom/__tests__/directives/vModel.spec.ts

```diff
@@ -681,6 +681,43 @@ describe('vModel', () => {
     expect(bar.selected).toEqual(true)
   })
 
+  it('v-model.number should work with select tag', async () => {
```

Comment:
Maybe should include a test case for multiple select also?

Reply:
Done.Thanks~😄